specials.lisp

```diff
@@ -78,6 +78,13 @@ variable has a true value, though.")
   "A `buffer' for one character.  Used by PEEK-CHAR* and
 UNREAD-CHAR*.")
 
+(defvar *max-line-length* nil
+  "Maximum length of a line.  If NIL, lines may be any length.")
```

Comment:
I'd appreciate more documentation. The docstring should address:
- what happens when attempting to read an overlong line.
- appropriate types and values this variable may take